src/main/java/Main.java

```diff
@@ -10,7 +11,22 @@ public static void main(String[] args) {
     }
 
     void run(Scanner scanner) {
-        // usupełnij metodę
+        Map<String, Country> countries = FileOperations.loadCountriesFromFile("src/main/countries.csv");
+        if (countries.isEmpty()) {
+            return;
+        }
+        System.out.println("Podaj kod kraju, o którym chcesz zobaczyć informacje:");
+        String code = scanner.nextLine();
+        for (Map.Entry<String, Country> entry : countries.entrySet()) {
+            if (entry.getKey().equals(code)) {
+                Country country = entry.getValue();
+                System.out.printf("%s (%s) ma %d ludności.", country.getCountryName(), country.getCountryCode(),
+                        country.getPeopleNumber());
+                return;
+            }
+
+        }
```

Comment:
Klucz w mapie to kod kraju. Ta pętla jest niepotrzebna, wystarczy metoda get(), która zwróci obiekt reprezentujący kraj, albo null